### PR TITLE
Duane/docker

### PR DESCRIPTION
**Pull request by michaeljvaldes**

### CODE REVIEW

**Review by AxiomaticAdi — APPROVED** [review]
image

Ask a developer to review 10 lines of code, and they will find 10 issues. Ask them to review 1000, and it looks good ship it

File: .env [hunk 1]
```diff
@@ -0,0 +1,9 @@
+DEBUG=1
+SECRET_KEY='7*fsmud8l*&)o^m2jp4$fn%nk$g)xh9z$y+q96_uj_+vmvirdj'
```
Comment (davidpchi) [inline .env:2]: lol is this a real secret key. i assume all of these things are for local .env

File: .env.dev [hunk 2]
```diff
@@ -0,0 +1,8 @@
+DEBUG=1
+SECRET_KEY='7*fsmud8l*&)o^m2jp4$fn%nk$g)xh9z$y+q96_uj_+vmvirdj'
+DJANGO_ALLOWED_HOSTS=localhost 127.0.0.1 [::1]
+POSTGRES_HOST=db
+POSTGRES_PORT=5432
+POSTGRES_DB=arcanis_dev
+POSTGRES_USER=arcanis_user
+POSTGRES_PASSWORD=arcanis_password
```
Comment (davidpchi) [inline .env.dev:8]: same comment here-- what's the difference between this .env and the other one

**Review by davidpchi — APPROVED** [review]
approve with suggestions

File: .env (hunk 1 above)
Comment (michaeljvaldes) [reply .env:2]: Yep, just for local dev

File: .env.dev (hunk 2 above)
Comment (michaeljvaldes) [reply .env.dev:8]: Just for local dev -- one is actually for running the server in docker, and the other is for running the server on the local host machine. These might change a lot by the time I'm done, so I haven't thought about whether I want to keep both yet tbh.

**Pull request merged**